packages/agreement-email-sender/src/index.ts

```diff
@@ -21,22 +18,26 @@ import {
   sendAgreementActivationEmail,
   senderAgreementSubmissionEmail,
 } from "./services/agreementEmailSenderService.js";
-import {
-  pecEmailManagerConfig,
-  emailManagerConfig,
-  agreementEmailSenderConfig,
-} from "./utilities/config.js";
 
-const config = agreementEmailSenderConfig();
-const kafkaConfig = kafkaConsumerConfig();
-const readModelConfig = readModelWriterConfig();
-const topicsConfig = agreementTopicConfig();
-const pecEmailConfig = pecEmailManagerConfig();
-const pecEmailManager = initEmailManager(pecEmailConfig);
-const emailConfig = emailManagerConfig();
-const emailManager = initEmailManager(emailConfig);
+import { config } from "./config/config.js";
+
+const emailManager = initEmailManager({
+  smtpAddress: config.smtpAddress,
+  smtpPort: config.smtpPort,
+  smtpSecure: config.smtpSecure,
+  smtpUsername: config.smtpUsername,
+  smtpPassword: config.smtpPassword,
+});
+const pecEmailManager = initEmailManager({
+  smtpAddress: config.pecSmtpAddress,
+  smtpPort: config.pecSmtpPort,
+  smtpSecure: config.pecSmtpSecure,
+  smtpUsername: config.pecSmtpUsername,
+  smtpPassword: config.pecSmtpPassword,
+});
+
```

Comment:
Merged from `main` after the PR was stale for a few weeks. @Carminepo2 I also adapted it after the changes introduced in #686, could you review this part and related changes in `agreement-email-sender` specifically?